test_schemas/en/test_new_routing_not.json

```diff
@@ -0,0 +1,177 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "language": "en",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.3",
+    "survey_id": "001",
+    "title": "Test Routing Not",
+    "theme": "default",
+    "description": "A test survey for routing based on the not operator",
+    "metadata": [
+        {
+            "name": "user_id",
+            "type": "string"
+        },
+        {
+            "name": "period_id",
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        }
+    ],
+    "questionnaire_flow": {
+        "type": "Linear",
+        "options": {
+            "summary": {
+                "collapsible": false
+            }
+        }
+    },
+    "sections": [
+        {
+            "id": "default-section",
+            "groups": [
+                {
+                    "blocks": [
+                        {
+                            "type": "Question",
+                            "id": "number-question-1",
+                            "question": {
+                                "answers": [
+                                    {
+                                        "id": "answer-1",
+                                        "mandatory": true,
+                                        "type": "Number",
+                                        "label": "Do not enter 123"
+                                    }
+                                ],
+                                "id": "question-1",
+                                "title": "Do not enter the number 123",
+                                "type": "General"
+                            }
+                        },
+                        {
+                            "type": "Question",
+                            "id": "number-question-2",
+                            "question": {
+                                "answers": [
+                                    {
+                                        "id": "answer-2",
+                                        "mandatory": true,
+                                        "type": "Number",
+                                        "label": "Do not enter 321"
+                                    }
+                                ],
+                                "id": "question-2",
+                                "title": "Do not enter the number 321",
+                                "type": "General"
+                            },
+                            "routing_rules": [
+                                {
+                                    "block": "correct-answer",
+                                    "when": {
+                                        "not": [
+                                            {
+                                                "and": [
+                                                    {
+                                                        "==": [
```

Comment:
Won't block since it technically demonstrates the feature, but I think we should have chosen a use case where `not` had to be used. Cases like this we want to have linting for in future, for example, instead of doing `not and ==`, we would do `or !=`. But use `not` for cases we cant avoid it, i.e `answer not in checkbox_answer_list`. In most cases, we would use `not` with an array operator. Although not technically not wrong to do `not x > 2` we should always use `x < 3` etc.

Reply:
I've changed this now so am using testing not with `in` rather than `and`